src/scripts/modules/article-aside.js

```diff
@@ -3,9 +3,7 @@ import headerComponent from './header.js'
 function init() {
   const articleAside =  document.querySelector('.article__aside')
 
-  if (!articleAside && !headerComponent) {
-    return
-  }
+  if (!articleAside) return
```

Comment:
Кажется, достаточно поправить здесь

```suggestion
if (!(articleAside && headerComponent)) {
```

Reply:
Ну Я не нашел упоминаний headerComponent в импортируемом файле, но можно и так